public/core.d.ts

```diff
@@ -8808,48 +8808,942 @@ declare namespace JMap {
    * Here you'll find all form related methods
    */
   namespace Form {
+
+    /**
+     * ***JMap.Form.getFormsMetaDataByLayerId***
+     * 
+     * Returns all forms metadata for a given layer id.
+     * 
+     * Fecth data from server first time, then keeps it in cache for the next time.
```

Comment:
**_Fecthes_** data from server _**the**_ first time, then keeps it in cache for the next time.